docs/source/examples/tex.rst

```diff
@@ -25,50 +77,62 @@ LaTeX
+parts either by their index (like ``tex[1]``), or you can look them up by (parts of) the tex code like
+in this example where we set the color of the ``\bigstar`` using :func:`~.set_color_by_tex`:
 
 .. manim:: Example4LaTeX
     :save_last_frame:
 
     class Example4LaTeX(Scene):
         def construct(self):
-            tex = Tex('Hello', '$\\bigstar$', '\LaTeX').scale(3)
+            tex = Tex('Hello', r'$\bigstar$', r'\LaTeX').scale(3)
             tex.set_color_by_tex('igsta', RED)
             self.add(tex)
 
+LaTeX Maths Fonts - The Template Library
+++++++++++++++++++++++++++++++++++++++++++++
+Changing fonts in LaTeX when typesetting mathematical formulae is a little bit more tricky than 
+with regular text. It requires changing the template that is used to compile the tex code.
+Manim comes with a collection of :class:`~.TexFontTemplates` ready for you to use. These templates will all work
+in maths mode:
+
 .. manim:: Example5LaTeX
     :save_last_frame:
 
     class Example5LaTeX(Scene):
         def construct(self):
-            tex = Tex('Hello \LaTeX', tex_template=TexFontTemplates.french_cursive).scale(3)
+            tex = Tex(r'$f: A \rightarrow B$', tex_template=TexFontTemplates.french_cursive).scale(3)
             self.add(tex)
 
-.. manim:: Example5bLaTeX
-    :save_last_frame:
-
-    class Example5bLaTeX(Scene):
-        def construct(self):
-            templateForFrenchCursive = TexTemplate(
-                preamble=r"""
-    \usepackage[english]{babel}
-    \usepackage{amsmath}
-    \usepackage{amssymb}
-    \usepackage[T1]{fontenc}
-    \usepackage[default]{frcursive}
-    \usepackage[eulergreek,noplusnominus,noequal,nohbar,nolessnomore,noasterisk]{mathastext}
-    """)
-            tex = Tex('Hello \LaTeX', tex_template=templateForFrenchCursive).scale(3)
-            self.add(tex)
+Manim also has a :class:`~.TexTemplateLibrary` containing the TeX templates used by 3Blue1Brown. One example 
+is the ctex template, used for typesetting Chinese. For this to work, the ctex LaTeX package
+must be installed on your system. Furthermore, if you are only typesetting Text, you probably do not
+need :class:`~.Tex` at all, and should use :class:`~.Text` or :class:`~.PangoText` instead. 
 
 .. manim:: Example6LaTeX
     :save_last_frame:
 
     class Example6LaTeX(Scene):
         def construct(self):
-            tex = Tex('Hello 你好 \LaTeX', tex_template=TexTemplateLibrary.ctex).scale(3)
+            tex = Tex('Hello 你好 \\LaTeX', tex_template=TexTemplateLibrary.ctex).scale(3)
             self.add(tex)
 
 
+Aligning formulae
+++++++++++++++++++
+A :class:`~.MathTex` mobject is typeset in the LaTeX  ``align*`` environment. This means you can use the ``&`` alignment
+character when typesetting multiline formulae:
+
+.. manim:: Example7LaTeX
+    :save_last_frame:
+
+    class Example7LaTeX(Scene):
+        def construct(self):
+            tex = MathTex(r'f(x) &= 3 + 2 + 1\\ &= 5 + 1 \\ &= 6').scale(2)
+            self.add(tex)
```

Comment:
Looks great!
I think it could also go to https://manimce--558.org.readthedocs.build/en/558/examples/formulas.html

Reply:
yes. eventually latex and formulas should be merged into just Tex-mobject documentation...